engine/packages/pegboard-gateway/src/lib.rs

```diff
@@ -541,4 +550,65 @@ impl CustomServeTrait for PegboardGateway {
 			Err(err) => Err(err),
 		}
 	}
+
+	#[tracing::instrument(skip_all, fields(actor_id=?self.actor_id))]
+	async fn handle_websocket_hibernation(&self, client_ws: WebSocketHandle) -> Result<bool> {
```

Comment:
make this return an enum instead of a bool for better clarity of what is going on